### ISSUE TITLE
Improve Python Colors

### ISSUE THREAD

**Issue by lovetoburnswhen** [mariorodeghiero/vue-theme-vscode#11]
A few scopes I noticed were missing:
- `keyword.control.import`
- `string.quoted.docstring`

**Comment by nirfuzz** [comment]
Hey @mariorodeghiero - would you be able to promote it..?

**Comment by mariorodeghiero** [comment]
@nirfuzz and @lovetoburnswhen can you paste a python repository mentioning where the piece of code contains this? so I can follow it as an example. 
Also, feel free to send PR's with the changes if you want, I really need code contributors.

**Comment by mariorodeghiero** [comment]
✅  Added support on V2 branch, this issue will be closed soon.

**Comment by mariorodeghiero** [comment]
@nirfuzz @lovetoburnswhen V2 is available, you can check the list of improvements [here v2](https://github.com/mariorodeghiero/vue-theme-vscode/releases/tag/2.0.0)
- keyword.control.import ✅ 

About `string.quoted.docstring` can you provide some python code that I can have a look at and try to improve the colors a bit? I will focus on the docstring now.

**Comment by nirfuzz** [comment]
<img width="396" alt="image" src="https://user-images.githubusercontent.com/7904773/193910394-bd56f6da-0521-4202-849b-19c42936ab58.png">
looks great! thanks @mariorodeghiero

**Comment by nirfuzz** [comment]
imports look great, though now there seems to be some inconsistency in string highlighting (not only docstring):

<img width="433" alt="image" src="https://user-images.githubusercontent.com/7904773/193933382-00a106cf-b348-4f3e-8881-bb5c2ae36128.png">

and another inconsistency with modules / methods:
<img width="375" alt="image" src="https://user-images.githubusercontent.com/7904773/193934494-e306350e-8de6-4455-b276-a3de4320e99f.png">

plus - no different colors for nested parentheses.


@mariorodeghiero

**Comment by mariorodeghiero** [comment]
@nirfuzz thanks for letting me know, I will check it.

**Comment by mariorodeghiero** [comment]
@nirfuzz I pushed a v2.0.1 with some Python support.

I will try to review a bit more once I have some time available(Now I am focused to improve the current colors) but for a while I think is ok. I just pull an open-source python project and I added some support.

Note: about no different colors for nested parentheses, I think is better that way for working with javascript, I tried some colors but I didn't like them.

![Screenshot 2022-10-12 at 22 07 39](https://user-images.githubusercontent.com/24671133/195438073-5d5942fb-3f32-4cf1-9e94-314439cff8d7.png)
